### PR TITLE
[zh] Fix secrets-good-practices page render error

### PR DESCRIPTION
**Pull request by wynterwang**

### CODE REVIEW

**Comment by howieyuen** [comment]
there is still an error on this page:

image

I check the translation format, and result is ok, don't know why...
PTAL: @tengqm

File: content/zh-cn/docs/concepts/security/secrets-good-practices.md [hunk 1]
```diff
@@ -18,15 +18,16 @@ weight: 70
 <!--
 {{<glossary_definition prepend="In Kubernetes, a Secret is an object that"
 term_id="secret" length="all">}}
+-->
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/security/secrets-good-practices.md:21]:
```suggestion
```
Line 18 - 21 need be deleted. The glossary block might be rendered twice.

**Review by windsonsea — COMMENTED** [review]
Preview FYI:
https://deploy-preview-37468--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/security/secrets-good-practices/

**Comment by windsonsea** [comment]
The preview looks good.

As the first PR, this is a good catch.

/lgtm

**Review by howieyuen — COMMENTED** [review]
/approve

**Pull request merged**